fire/cli/mtl.py

```diff
@@ -31,7 +31,7 @@
 # Datahåndtering
 import pandas as pd
 import xlsxwriter
-from datetime import date, time
+from datetime import datetime
```

Comment:
Det er virkelig dumt at modulet `datetime` indeholder klassen `datetime`... Det ses også ret ofte at `datetime`, altså modulet, omdøbes til `dt`: 
``` 
import datetime as dt
```

Så spilder man ikke så mange tegn og det er stadig til at forstå hvor tingene kommer fra. Når nu man ikke har sørget for at de relevante klasser hedder `datetime.DateTime`, `datetime.Date`, `datetime.Time` som ellers foreskrevet af Pythons egen styleguide.

Bare et rant, ikke et ændringsønske.

Reply:
Tak for review, i øvrigt.

Jeg merger denne, på trods af fail på manglende test coverage: Mit næste PR handler om test, og jeg skal lige have denne på plads inden Tanya tager over igen efter klokken 14: Da det bliver første testelement her, så bliver næste PR ret omfattende (for mig, mentalt i hvert fald - der er en del der både skal forstås og sættes sammen og på plads)

Reply:
Jeg har ikke helt luret hvordan codecov håndterer fejlmelding på test coverage. Der burde være lidt slæk i systemet, sådan at ubetydelige ændringer ikke ender i en fejl. Jeg fikser det når jeg lige får tid til det.